sdk/state/update_test.go

```diff
@@ -38,8 +38,12 @@ func TestLastConfirmedPayment(t *testing.T) {
 		RemoteEscrowAccount: localEscrowAccount,
 	})
 
+	// latest close agreement should be set during open steps
```

Comment:
this would normally be set during the `open` tests. Mimicking that for this unit test